test/parse-outputs/test_parse_outputs.py

```diff
@@ -87,7 +87,7 @@ def test_duplicate_edges(self):
 
     def test_empty_raw_pathway(self):
         for algo, params in algorithms.items():
-            files = glob(str(INDIR / "empty" / f"{algo}-empty-raw-pathway*"))
+            files = glob(str(INDIR / "empty" / f"{algo}-empty-pathway*"))
```

Comment:
This was correct before, for the out_file it should be changed to f"{algo}-empty-pathway-output.txt"

Reply:
The files for the input are the empty-raw-pathway files